Operators_and_constructions.rb

```diff
@@ -0,0 +1,51 @@
+# 1. Create method which gets 10 random integers (from 0 to 9) (generate it using while or until)
+# before output for each next number we should puts "========="
+# if number is even or odd it should puts string "the number is even" or "the number is odd"
+# if it's more (or less) than 5 it should puts appropriate string "the number is more (or less) than 5"
+# in case we've got "0" we should puts "we've got 0"
+# in each case we should get 2 messages except the '0' case
+
+
+
+def gets_random_10
+
+  count = 0
+
+while count < 10
+  num = rand(9)
+ first_string = "=========" + " #{num}"
```

Comment:
this line with equal signs is to more clear visual experience
so first line is only '========='
and then other lines :)